client/src/styles/App.css

```diff
@@ -1,3 +1,10 @@
+:root {
+  --colNum: 3;
+  --colNum: 3;
```

Comment:
`colNum` doesn't really tell me what this is. Maybe `colCount` to suggest it's a count of the number of columns?
(Also that second one should be `rowNum`/`rowCount`. 😉)